bluesky/run_engine.py

```diff
@@ -228,6 +228,17 @@ def __init__(self, md=None, *, loop=None, preprocessors=None,
         if md is None:
             md = {}
         self.md = md
+
+        try:
+            import ophyd
+            self.md['OPHYD_VERSION'] = ophyd.__version__
+        except ImportError:
+            self.log.exception("Failed to import __version__ from ophyd.")
```

Comment:
In my opinion this is too aggressive. At most this should log at DEBUG level. If I'm using the RunEngine without ophyd, I don't want to see loud ERROR-level tracebacks.

Reply:
updated